README.Rmd

```diff
@@ -188,15 +188,15 @@ irm("foo.csv", trash = FALSE)
 ils()
 ```
 
-<!-- The user Bobby can also be removed again. -->
+<!-- The user Alice can also be removed again. -->
 
-```{r rmbobby, include=FALSE}
+```{r rmAlice, include=FALSE}
 # authenticate
 iauth("rods", "rods")
 
-# remove user bobby
-rirods:::iadmin(action = "remove", target = "user", arg2 = "bobby")
+# remove user Alice
+rirods:::iadmin(action = "remove", target = "user", arg2 = "Alice")
```

Comment:
"Alice" -> "alice"

might consider always using lowercase EVERYWHERE in this readme - 'Alice' and 'alice' are different users in iRODS (it is case-sensitive)